mlir/lib/Conversion/GPUToSPIRV/GPUToSPIRVPass.cpp

```diff
@@ -16,6 +16,7 @@
 #include "mlir/Conversion/ArithToSPIRV/ArithToSPIRV.h"
 #include "mlir/Conversion/FuncToSPIRV/FuncToSPIRV.h"
 #include "mlir/Conversion/GPUToSPIRV/GPUToSPIRV.h"
+#include "mlir/Conversion/VectorToSPIRV/VectorToSPIRV.h"
```

Comment:
Does this require cmake/bazel dependencies to be updated?

Reply:
I didn't update cmake/bazel, and didn't run into problem when rebuilding

Reply:
These are often 'masked' with cmake -- they happen to work until you switch build options to shared libs or build with bazel. You can confirm by running the missing deps ninja tool: https://discourse.llvm.org/t/ninja-can-now-check-for-missing-cmake-dependencies-on-generated-files/74344

Reply:
I ran the ninja tool and did not see this file in the output